backend/tests/routes/test_users.py

```diff
@@ -1,13 +1,12 @@
 # TODO: re-enable when we have account updated email
 # from unittest.mock import patch
 
-from app.db.models import User
 
-
-def test_authenticated_user_me(client, user_token_headers):
-    response = client.get("/api/v1/users/me", headers=user_token_headers)
-    assert response.status_code == 200
-    assert response.headers.get("Cache-Control") == "no-cache, no-store, private"
+def test_authenticated_user_me(client):
```

Comment:
Should we just remove this file?

Reply:
I'm not sure if we're going to want these pages for admin users... might be safe to remove them & re-add later if we decide we want them...

Reply:
removed